src/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 cmake_minimum_required(VERSION 3.13)

```

Comment:
I think we need to bump this to 3.14 to ensure FetchContent is available.

Reply:
While 3.14 would be preferred (it adds the ability for FetchContent to access the terminal to do password prompts and whatnot), FetchContent was technically added in 3.11 so I wasn't sure if a bump was actually needed. Should I just go ahead and bump it?

Reply:
`FetchContent_MakeAvailable` is 3.14. Before that, only `_Declare` and `_Populate` were there.

Reply:
Oh I'd totally missed that. Fair enough then, will bump :)